src/main/java/net/dries007/holoInventory/util/Helper.java

```diff
@@ -31,11 +31,13 @@
 import net.minecraft.util.MovingObjectPosition;
 import net.minecraft.util.Vec3;
 
+import appeng.tile.misc.TileInterface;
+
 public class Helper
 {
     public static boolean weWant(Object o)
     {
-        return o != null && (o instanceof IInventory || o instanceof IMerchant || o instanceof TileEntityEnderChest || o instanceof BlockJukebox.TileEntityJukebox);
+        return o != null && (o instanceof IInventory || o instanceof IMerchant || o instanceof TileEntityEnderChest || o instanceof BlockJukebox.TileEntityJukebox || o instanceof TileInterface);
```

Comment:
Does `IPartHost` also need to be added here?

Reply:
Indeed, I wonder how it worked in my test then, perhaps TileCableBus injects IInventory into itself on part placement. AE multipart tile is black magic.

Reply:
Huh, weird. But yeah, probably better to be safe.